.circleci/config.yml

```diff
@@ -2,7 +2,8 @@ version: 2.1
 jobs:
   build-and-test:
     macos:
-      xcode: 15.0.0
+      xcode: 15.0.1
```

Comment:
Can this be specified less specific? Having to update for each patch version seems unnecessary. What about the resource_class, can that be more general?

Reply:
Was trying to check in their docs, but found no example of how to be less specific, but could of course just try to say 15 and see what happens 🤷

Reply:
It works for Travis to use just 15 or 15.0, we should try same here...

Reply:
> It works for Travis to use just 15 or 15.0, we should try same here...

They are two completely different services though..

> Was trying to check in their docs, but found no example of how to be less specific, but could of course just try to say 15 and see what happens 🤷

They specify `15.0.0` in their docs https://circleci.com/docs/testing-ios/  but might just default to `15`

Reply:
For sure